packages/ui/aurora/package.json

```diff
@@ -40,6 +41,7 @@
     "@radix-ui/react-toolbar": "^1.0.3",
     "@radix-ui/react-tooltip": "^1.0.5",
     "@radix-ui/react-use-controllable-state": "^1.0.0",
+    "downshift": "^8.1.0",
```

Comment:
In principle we create a separate package when a component has special dependencies like this.

Reply:
Or, this should be a primitive first, then themed in Aurora.